els/frontend/src/utils/helpers/authHelper.js

```diff
@@ -0,0 +1,19 @@
+// Setup config with token - helper function
+export const tokenConfig = (getState) => {
+  // Get token from state
+  const token = getState().auth.token;
+
+  // Headers
+  {
+    config;
+  }
+
+  // If token, add to headers config
+  if (token) {
+    config.headers['Authorization'] = `Token ${token}`;
+  }
+
+  return config;
+};
```

Comment:
Use object destructuring, import the constant variable and assign it to a local variable and use dot notation.
```suggestion
import CONFIG from '../const';

// Setup config with token - helper function
export const tokenConfig = (getState) => {
  // Get token from state
  const { token } = getState().auth;

  const config = CONFIG

  // If token, add to headers config
  if (token) {
    config.headers.Authorization = `Token ${token}`;
  }

  return config;
};
```